api/v1beta1/octavia_webhook.go

```diff
@@ -198,9 +198,34 @@ func (r *Octavia) ValidateDelete() (admission.Warnings, error) {
 	return nil, nil
 }
 
-func (spec *OctaviaSpec) GetDefaultRouteAnnotations() (annotations map[string]string) {
-	annotations = map[string]string{
+func (spec *OctaviaAPISpecCore) GetDefaultRouteAnnotations() (annotations map[string]string) {
```

Comment:
As per my comment on the other PR:
https://github.com/openstack-k8s-operators/openstack-operator/pull/1071

Did you mean to add this to the `OctaviaSpecCore`:
```suggestion
func (spec *OctaviaSpecCore) GetDefaultRouteAnnotations() (annotations map[string]string) {
```

If I make that change, and then check it again, then it works.